index.js

```diff
@@ -61,3 +60,16 @@ function isEnclosure(str) {
 
   return str.slice(foundIndex + 1, -1).includes(slash);
 }
+
+function isGlobby(str) {
+  if (/\([^()]+$/.test(str))  {
+    return true;
+  }
+  if (str[0] === '{' || str[0] === '[') {
+    return true;
+  }
+  if (/[^\\][{[]/.test(str)) {
+    return true;
+  }
+  return isGlob(str);
```

Comment:
Is there anything weird (different?) by changing the order of the isGlob check and "globby" regexp test?

Reply:
Because they were OR'ed, I don't think it makes any difference logically. I'm happy to flip it back assuming that doesn't introduce performance regressions.